scrapy/commands/genspider.py

```diff
@@ -61,6 +62,12 @@ def run(self, args, opts):
             raise UsageError()
 
         name, domain = args[0:2]
+
+        parsed_url = urlparse(domain)
```

Comment:
```suggestion
        parsed_url = urlparse(domain.rstrip('/'))
```
Perhaps not strictly necessary, but if I'm not mistaken an URL that ends with a slash would result in two slashes when it's used later in the template (`$url/feed.xml` -> `https://example.org//feed.xml` if `$url == https://example.org/`)